installation/routines/setup_jukebox_core.sh

```diff
@@ -99,7 +100,7 @@ _jukebox_core_build_and_install_pyzmq() {
     fi
 
     sudo ZMQ_PREFIX="${ZMQ_PREFIX}" ZMQ_DRAFT_API=1 \
-         pip3 install --no-cache-dir --no-binary "pyzmq" --pre pyzmq
+         pip3 install --no-cache-dir --no-binary "pyzmq" --pre --break-system-packages pyzmq
```

Comment:
I think you should set this globally (which is also backwards compatible to bullseye and buster): `sudo python3 -m pip config set global.break-system-packages true`

Reply:
hey @s-martin! thanks for the feedback! do you really want to modify the global settings for the OS? I personally would not like to do that as this affects other software on the system as well. We can easily get the OS via something like `lsb_release -a | grep -i "Codename:" | cut -f2`. If that reveals something like `bullseye` or `buster`, we can then avoid the `break-system-packages` option. what do you think?

Reply:
The config setting is set for the user space, not the entire OS.
But if set as config or optional parameter is has the same effect: the packages will be installed on the system and not in venv (so could break things).

On the long run we should properly implement a venv, but that's nothing quick done. So the suggestion of @s-martin would be the most easiest solution.

Reply:
alright @s-martin and @AlvinSchiller